<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DevFrank实用工具箱</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbShare">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>760</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>分享给好友</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>810</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;© 2024 DevFrank. 欢迎提出宝贵建议&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>581</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pageHome">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>541</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>开发运维</string>
      </property>
      <widget class="QPushButton" name="pbTimestamp">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>190</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>时间戳转换</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbUuid">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>190</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>UUID生成器</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>541</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>地理工具</string>
      </property>
      <widget class="QPushButton" name="pbMap">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>190</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>地图信息</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbWeather">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>190</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>天气信息</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>541</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>生活工具</string>
      </property>
      <widget class="QPushButton" name="pbBmi">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>190</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>BMI指数</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbCount">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>190</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>倒数日</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>系统工具</string>
      </property>
      <widget class="QPushButton" name="pbDevice">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>190</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>设备信息</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbClock">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>190</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>全屏时钟</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="pageDevice">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>设备信息</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="tbOS">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>401</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbHome">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>760</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>返回主页</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
